projects/CMRxRecon/tools/create_symlinks.py

```diff
@@ -0,0 +1,49 @@
+import glob
+import os
+import pathlib
+from typing import Union
+
+
+def create_symlinks(base_path: Union[str, pathlib.Path], sym_base_path: Union[str, pathlib.Path], prefix: str = ""):
+    """Creates symlinks of data from different directories data in a single directory.
+
+    Paths should be provided in absolute form.
+
+    Parameters
+    ----------
+    base_path : Union[str, pathlib.Path],
+        Must point to directory with structure:
+        Cine/Mapping
+        - Type of Data
+        -- P001
+        -- P002
+    sym_base_path : Union[str, pathlib.Path],
+        Path to save symbolic data.
+    prefix : str
+        Prefix for symlink data.
+    """
+
+    base_path = pathlib.Path(base_path)
+
+    sym_base_path = pathlib.Path(sym_base_path)
+
+    if not os.path.exists(sym_base_path):
+        print(f"Creating symbolic path: {sym_base_path}...")
+        os.makedirs(sym_base_path)
+
```

Comment:
Prints are not too nice, can't you use a logger?

Also use pathlib instead of os.path.

Reply:
Done and done